### DISCUSSION TITLE
Neume Paragraphs / Handling neume insert between text boxes

### DISCUSSION THREAD

**Discussion by Beatl3** [neanes/neanes#462] [Ideas]
I have trouble moving things around. For example now I have written a text box and a music text paragraph under. I am trying to add a line of new music text under the existing textbox (to start a new music text paragraph above the existing one) but I cannot find an easy way  doing that. A button for adding a new empty music line after an existing textbox would be very desirable.

**Comment by danielgarthur** [comment]
I'm not sure whether I am understanding. There are line break buttons that let you add a new line. Is that what you are looking for?
<img width="680" alt="image" src="https://user-images.githubusercontent.com/4132779/220183881-d0e7827f-3469-4687-abf3-b21d27dbaeb0.png">

**Comment by danielgarthur** [comment]
Reading your message again, I think maybe you were actually talking about this situation?
<img width="512" alt="image" src="https://user-images.githubusercontent.com/4132779/220184988-4a8bd4c6-faf3-4c6f-891b-c6e54bcdacd5.png">

I've never considered this situation before. You can certainly add neumes between the boxes using the insert mode (see screenshot below), but the behavior is kind of odd. I'd have to think more about how this could be improved.

<img width="123" alt="image" src="https://user-images.githubusercontent.com/4132779/220185106-8b139896-40ae-415e-a5cb-77b0cfad9705.png">

Right now, there is no concept of a "neume paragraph". I think something like that might help.

**Comment by Beatl3** [comment]
Thank you, using this work around could help for now for forcing new neume paragraphs, i.e., adding a new text box and then adding a neume after that.

**Comment by danielgarthur** [comment]
If you add a right-aligned martyria, then the inserted neumes will be formatted better (i.e. not all spaced out).

After
<img width="493" alt="image" src="https://user-images.githubusercontent.com/4132779/220191992-cb8a33e0-8cee-43e9-ad20-96370a133cfc.png">

Before
<img width="487" alt="image" src="https://user-images.githubusercontent.com/4132779/220192039-bde00796-a5f4-4eee-8683-e362b5c98983.png">

**Comment by Beatl3** [comment]
Right, that I have found out :-)

**Comment by danielgarthur** [comment]
See #1126.
